java/runtime-native/src/main/java/org/ray/runner/RunManager.java

```diff
@@ -48,7 +48,7 @@ public RunManager(RayParameters params, PathConfig paths, ConfigReader configRea
 
   private static boolean killProcess(Process p) {
     if (p.isAlive()) {
-      p.destroyForcibly();
+      p.destroy();
```

Comment:
Does this change fix any issue?

Reply:
Many backend processes are exited by a `SIGTERM` signal. When they do some tail-in works when receive the signal.
If we use `destroyForcibly()`, the backend processes will be killed without any signal. Therefore, the backend cannot be exited safely and do some tail-in works.